src/mantle/framework/console/class-test-config-install-command.php

```diff
@@ -68,7 +68,13 @@ public function handle( array $args, array $assoc_args = [] ) {
 			$this->error( __( 'Error copying configuration file.', 'mantle' ), true );
 		}
 
-		$this->log( __( 'Configuration copied! Update it to reference your proper database credentials.', 'mantle' ) );
+		$this->log(
+			sprintf(
+				$path,
+				/* translators: 1: config file path */
+				__( 'Configuration copied to %s. Update it to reference your proper database credentials.', 'mantle' ),
+			),
+		);
```

Comment:
I think these parameters are reversed, I'm assuming `$path` is meant to satisfy the `%s`?